extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -289,6 +294,11 @@ Here's an example of the value of the `makePaymentRequest` custom field, with ge
     "encryptedExpiryYear": "test_2030",
     "encryptedSecurityCode": "test_737"
   },
+  "additionalData": {
+    "allow3DS2": true
+  },
+  "channel": "Web",
```

Comment:
I am not sure if it is case sensitive or not in Adyen. However, I've used lowercase - `web` and I can confirm this case works fine - not sure about `Web`

Reply:
If you decide to change it, please remember there are 3 places below to update too

Reply:
It's "Web" in the Adyen docs, probably ok to keep as is.